src/pybind/mgr/dashboard/frontend/src/app/ceph/rgw/rgw-multisite-export/rgw-multisite-export.component.html

```diff
@@ -1,69 +1,72 @@
-          </label>
-          <div class="cd-col-form-input">
-            <input id="realmName"
-                   name="realmName"
-                   type="text"
-                   value="{{ realminfo.realm }}"
-                   readonly>
-          </div>
+
+      <div *ngIf="!loading">
+        <div class="form-item">
+          <cd-alert-panel *ngIf="!tokenValid"
+                          type="warning"
+                          i18n>
+            <div *ngFor="let realminfo of realms">
+              <b>{{realminfo.realm}}</b> - {{realminfo.token}}
+            </div>
+          </cd-alert-panel>
         </div>
-        <div class="form-group row">
-          <label class="cd-col-form-label"
-                 for="token"
-                 i18n>Token
-          </label>
-          <div class="cd-col-form-input">
-            <input id="realmToken"
-                   name="realmToken"
-                   type="text"
-                   value="{{ realminfo.token }}"
-                   class="me-2 mb-4"
-                   readonly>
-            <cd-copy-2-clipboard-button
-                      source="{{ realminfo.token }}"
-                      [byId]="false">
-            </cd-copy-2-clipboard-button>
+
+        <div *ngFor="let realminfo of realms; let last = last">
+          <div class="form-item">
+            <cds-text-label labelInputID="realmName"
+                            i18n>Realm Name
+              <input cdsText
+                     readonly
+                     type="text"
+                     id="realmName"
+                     name="realmName"
+                     [value]="realminfo.realm" />
+            </cds-text-label>
           </div>
-          <hr *ngIf="realms.length > 1">
+
+          <div class="form-item">
+            <div class="cds-input-group">
+              <cds-text-label labelInputID="realmToken"
+                              i18n>Token
+                <input cdsText
+                       readonly
+                       type="text"
+                       id="realmToken"
+                       name="realmToken"
+                       [value]="realminfo.token" />
+              </cds-text-label>
+              <cd-copy-2-clipboard-button [source]="realminfo.token"
+                                          [byId]="false"
+                                          size="md"
+                                          class="clipboard">
+              </cd-copy-2-clipboard-button>
+            </div>
+          </div>
+          <hr *ngIf="!last" />
         </div>
       </div>
     </div>
-    <div class="modal-footer">
-      <cd-back-button class="m-2 float-end"
-                      aria-label="Close"
-                      (backAction)="activeModal.close()"></cd-back-button>
-    </div>
-    </form>
-  </ng-container>
-</cd-modal>
+    <cds-modal-footer>
+      <cd-back-button aria-label="Close"
+                      (backAction)="closeModal()"></cd-back-button>
```

Comment:
```suggestion
      <cd-form-button-panel [modalForm]="true"
                        [showSubmit]="false"
                        (backAction)="closeModal()"></cd-form-button-panel>
```
So that  can get back button consuming 50% width